src/helper.cc

```diff
@@ -1244,7 +1235,7 @@ Enqueue(Helper::Client * const hlp, Helper::Xaction * const r)
     /* do this first so idle=N has a chance to grow the child pool before it hits critical. */
     if (hlp->childs.needNew() > 0) {
         debugs(84, DBG_CRITICAL, "Starting new " << hlp->id_name << " helpers...");
```

Comment:
Please take advantage of the chance to drop this overly verbose `debugs()` - better info is displayed by the `openSessions()` methods.

```suggestion
```